spec/unit/validator_spec.rb

```diff
@@ -487,6 +499,7 @@ def podspec_path
       end
 
       def setup_validator
+        @validator.instance_variable_set(:@results, [])
```

Comment:
Because the variable is initialized in `validate`, which is not called directly in the tests, but instead private methods, still there are side-effects appearing on it.